src/components/partials/MapInfoWindow.tsx

```diff
@@ -0,0 +1,29 @@
+import Box from '@mui/material/Box'
+import Stack from '@mui/material/Stack'
+import { Link } from 'react-router-dom'
+import { Restaurant } from '../../client/flavorite/models'
+
+type infoWindowProps = {
+  restaurants: Restaurant[]
+  coords: { lat: number; lng: number }
+}
+
+export default function MapInfoWindow({ restaurants, coords }: infoWindowProps) {
+  const displayInfoWindow = restaurants.map((restaurant, idx) => {
+    if (restaurant.latitude === coords.lat && restaurant.longitude === coords.lng) {
```

Comment:
could be better to filter the restaurants with the coord and map.
restaurants.filter(...).map(...)

Reply:
yup thanks. Edited to:
```const displayInfoWindow = restaurants
    .filter(
      (restaurant) => restaurant.latitude === coords.lat && restaurant.longitude === coords.lng,
    )
    .map((restaurant, idx) => {...}```